SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp

```diff
@@ -66,6 +66,18 @@ FColor USpatialStatics::GetInspectorColorForWorkerName(const FString& WorkerName
 	return SpatialGDK::GetColorForWorkerName(WorkerName);
 }
 
+bool USpatialStatics::IsSpatialMultiWorkerEnabled(const UWorld* World)
```

Comment:
As this is a blueprint function, perhaps just use the WorldContextActor, to make it easier on the blueprint user.